day3/src/actions/index.js

```diff
@@ -0,0 +1,33 @@
+export const CARD_FLIPPED = 'CARD_FLIPPED';
```

Comment:
På UiO fikk jeg inntrykk av at intro til funksjonell programmering og basic react var helt nytt for de aller fleste. Mulig det er noe annerledes på NTNU, men tror nok det er viktig uansett hvordan vi velger å gjøre det, at vi forklarer det grunnleggende tydelig nok, slik at vi ikke mister de helt nybegynnerne.. Er nok neppe mange av dem som har mye erfaring med React og Redux. Og de som evt. har det, er kanskje ikke de som uansett ville lært mest av dette kurset.